<launch>
  <node name="product_detector" pkg="ahold_product_detection" type="detector.py" output="screen"/>
  <node name="product_detector" pkg="ahold_product_detection" type="detector_2.py" output="screen"/>
  <node name="product_pose_estimator" pkg="ahold_product_detection" type="pose_estimator.py" output="screen"/>
  <node name="product_tracker" pkg="ahold_product_detection" type="tracker.py" output="screen"/>
</launch>